spec/classes/init_spec.rb

```diff
@@ -49,7 +49,7 @@
         it { is_expected.to contain_concat_fragment('squid_header').without_content(%r{^err_page_stylesheet}) }
       end
 
-      context 'with all parameters set' do
+      context 'with all parameters set' do # rubocop:todo RSpec/MultipleMemoizedHelpers
```

Comment:
I got hit by this a lot… Maybe we should consider increase the limit, or disable the test completely?

Reply:
Yes. 
I've tried re-arranging things to not get this elsewhere but don't quite understand how to avoid it.